gravity.sh

```diff
@@ -10,8 +10,24 @@
 # the Free Software Foundation, either version 2 of the License, or
 # (at your option) any later version.
 
+# Run this script as root or under sudo
+if [[ $EUID -eq 0 ]];then
+	echo "You are root."
+else
+	echo "::: sudo will be used for the install."
```

Comment:
Thanks for the submission, can you take this one line out, I think it may have been left over from the copy from `basic-install.sh`. It's out of place since `gravity.sh` doesn't install anything.